slices/erlang-asn1.yaml

```diff
@@ -0,0 +1,17 @@
+package: erlang-asn1
+
+essential:
+  - erlang-asn1_copyright
+
+slices:
+  libs:
+    essential:
+      - erlang-base_bins
```

Comment:
why do these libs need the erland-base_bins?

Reply:
see comment above.

Reply:
https://github.com/canonical/chisel-releases/pull/258#discussion_r1802799227